### DISCUSSION TITLE
Can somebody please fix these dictionary? The links within them don't work !Duden Redewendung (dsl) and Dornseiff (dsl)

### DISCUSSION THREAD

**Discussion by KamlaK10** [goldendict/goldendict#1631] [General]
hello,

Can somebody pls fix these dictionaries?
The links to these dictionaries.
https://disk.yandex.com/d/u7cBOXTzn9_p9w

They work fine in Goldendict fine but you often encounter the following problems. I cannot fix the described issue below because I don't know coding. I just use them as a layman to search words.

Whenever you search for a idiom in Duden Redewendung, it shows like this
![1](https://user-images.githubusercontent.com/89730239/235484926-9484f15d-77fa-49a5-a2fd-9c010a0b8fd0.png)
Can it be like we could click this arrows and reach the definition?

Same is the case for this Dornseif 2020 dictionary
![3](https://user-images.githubusercontent.com/89730239/235485117-7d7d7cf2-6447-465b-8de9-04e69808e71e.png)
As seen in the picture above, although the links are clickable at first, but once you click on them the next set of links within a page are not clickable (shown in image below and encircled in green)

![2](https://user-images.githubusercontent.com/89730239/235485194-baddb406-d0dc-42ef-a28b-3deb5b0818f5.png)
Could it be that the user could also click on the list of names encircled above in green ?

Sorry if that is too much of an asking, but I don't know if this is time-consuming or not?!

**Comment by vedgy** [comment]
We'd have to figure out whether this is a GoldenDict bug, in which case the discussion has to be replaced with an issue; or broken dictionaries, in which case this can stay a discussion, because fixing the dictionaries themselves is not a GoldenDict developer's job. Have you tried loading these dictionaries into a different dictionary lookup program?
